okhttp/src/main/java/okhttp3/internal/concurrent/TaskRunner.kt

```diff
@@ -166,4 +166,8 @@ class TaskRunner(
       taskExecutor.shutdown()
     }
   }
+
+  companion object {
+    val INSTANCE = TaskRunner(RealBackend())
```

Comment:
If this INSTANCE gets shutdown via INSTANCE.backend.shutDown() are you happy with RejectedExecutionException being raised?  Or do you want a clearer message to educate the developer?  OR should this instance not allow shutdown?

Reply:
This one shouldn’t ever be shut down. It’s intended as an internal API but I’ve learned that sometimes developers can’t help themselves!